backend/services/graph_service.py

```
@@ -344,12 +393,12 @@ def apply_graph_update(user_id: str, graph_update: dict) -> list:
-    # Refresh shared course context for every subject touched in this update
-    if touched_subjects:
+    # Refresh shared course context for every course touched in this update
+    if touched_courses:
         from services.course_context_service import update_course_context
-        for subj in touched_subjects:
+        for cid in touched_courses:
             try:
-                update_course_context(subj)
+                update_course_context(cid)
             except Exception:
                 pass  # never block the main response for a context refresh
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)